metric-anomaly-data-model/src/main/avro/MetricAnomalyNotificationEvent.avdl

```diff
@@ -1,26 +1,38 @@
 @namespace("org.hypertrace.alert.engine.metric.anomaly.datamodel")
 protocol MetricAnomalyNotificationEventProtocol {
 
-  enum Operator {
+  enum StaticThresholdOperator {
     STATIC_THRESHOLD_OPERATOR_LTE,
     STATIC_THRESHOLD_OPERATOR_GTE,
     STATIC_THRESHOLD_OPERATOR_LT,
     STATIC_THRESHOLD_OPERATOR_GT
   }
 
+  record StaticRuleViolationSummary {
+    int dataCount;
+    int violationCount;
+    array<double> metricValues;
```

Comment:
We need to rethink that do we want to send all metric values. 
- would it be useful? 
   - if, we have the last 5 min, and aggregation at 15s, we will have 20 points. 
   - And, if aggregation is 30s, it will have 10 points. 
- So, if we will use this information, I guess somewhere, we have to show aggregation time used for metrics as well.
Anyway, once we have defaulted in all avdl, it will be easy to add/discard with compatibilty.

Reply:
Yeah, we can rethink this